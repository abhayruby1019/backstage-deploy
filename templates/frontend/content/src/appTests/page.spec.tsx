import { render } from '@testing-library/react'
import RootPage from '@/app/page'

describe('Home Page RSC', () => {
  it('should display technical error when type is not passed', () => {
    const { container } = render(<RootPage />)
    expect(container).toMatchSnapshot()
  })
})
