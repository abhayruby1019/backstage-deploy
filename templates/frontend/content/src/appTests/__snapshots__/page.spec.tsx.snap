// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page RSC should display technical error when type is not passed 1`] = `
<div>
  <main
    class="main"
  >
    <h1>
      Poonawala Fincop Ltd - demo
    </h1>
  </main>
</div>
`;
